_source/user-guide/kibana/index.md

```diff
@@ -8,24 +8,25 @@ tags:
   - kibana
 contributors:
   - imnotashrimp
+  - shalper
```

Comment:
@shalper What is the scope of this page? I feel there is a lot more we can say here

Reply:
Agreed. This is a quick fix just so the first page doesn't show an out-of-date image. Better intro is in process.